gui/wxpython/gui_core/dialogs.py

```diff
@@ -1213,8 +1213,8 @@ def _filter(self, data):
             try:
                 if re.compile(self.flt_pattern).search(dt):
                     flt_data.append(dt)
-            except:
-                pass
+            except re.error as e:
+                print(f"Error compiling regex pattern: {e}")
```

Comment:
The intention here was to ignore the error, so the printing is not needed. Using print for these situations in GUI is typically not appropriate anyway, a better solution would be to reraise the exception.

```suggestion
            except re.error:
                pass
```